utils/constants.py

```diff
@@ -44,3 +44,82 @@
     'os_version',
     'phone_lang',
 ]
+
+VALID_DEMOGRAPHICS_COLS = [
+    'user_id',
+    '_id',
+    'data.ts',
+    'data.jsonDocResponse.aSfdnWs9LE6q8YEF7u9n85.group_hg4zz25.What_is_your_race_ethnicity',
+    'data.jsonDocResponse.aSfdnWs9LE6q8YEF7u9n85.group_hg4zz25.Are_you_a_student',
+    'data.local_dt.year',
+    'data.local_dt.month',
+    'data.local_dt.day',
+    'data.local_dt.hour',
+    'data.local_dt.minute',
+    'data.local_dt.second',
+    'data.local_dt.weekday',
+    'data.local_dt.timezone',
+    'read_ts',
+    'metadata.time_zone',
+    'metadata.plugin',
+    'metadata.write_ts',
+    'metadata.platform',
+    'metadata.read_ts',
+    'metadata.key',
+    'metadata.type',
+    'metadata.write_local_dt.year',
+    'metadata.write_local_dt.month',
+    'metadata.write_local_dt.day',
+    'metadata.write_local_dt.hour', 
+    'metadata.write_local_dt.minute',
+    'metadata.write_local_dt.second',
+    'metadata.write_local_dt.weekday',
+    'metadata.write_local_dt.timezone',
+    'metadata.write_fmt_time',
+    'data.ts',
+    'data.xmlResponse',
+    'data.label',
+    'data.jsonDocResponse.aSfdnWs9LE6q8YEF7u9n85.attr.id',
+    'data.jsonDocResponse.aSfdnWs9LE6q8YEF7u9n85.attr.xmlns:jr',
+    'data.jsonDocResponse.aSfdnWs9LE6q8YEF7u9n85.attr.xmlns:orx',
+    'data.jsonDocResponse.aSfdnWs9LE6q8YEF7u9n85.group_yk8eb99.At_your_primary_job_do_you_ha',
+    'data.jsonDocResponse.aSfdnWs9LE6q8YEF7u9n85.group_yk8eb99.Which_best_describes_your_prim',
+    'data.jsonDocResponse.aSfdnWs9LE6q8YEF7u9n85.group_yk8eb99.Do_you_work_full_time_or_part_',
+    'data.jsonDocResponse.aSfdnWs9LE6q8YEF7u9n85.group_yk8eb99.Do_you_have_the_option_of_work',
+    'data.jsonDocResponse.aSfdnWs9LE6q8YEF7u9n85.group_yk8eb99.Please_describe_your_primary_job',
+    'data.jsonDocResponse.aSfdnWs9LE6q8YEF7u9n85.group_yk8eb99.Do_you_have_more_than_one_job',
+    'data.jsonDocResponse.aSfdnWs9LE6q8YEF7u9n85.group_yk8eb99.What_days_of_the_week_do_you_t',
+    'data.jsonDocResponse.aSfdnWs9LE6q8YEF7u9n85.group_yk8eb99.How_many_days_do_you_usually_w_001',
```

Comment:
We cannot hardcode these because the survey can be different for each partner.
You should exclude the entries that the config wants to be excluded (currently marked as TODO) and potentially exclude several of the entries here. But we should not have a list of valid cols